releasenotes/notes/wait-when-distribution-tracing-disabled.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+releaseNotes:
+  - |
+    **Fixed** `x wait` has undecipherable message when PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING is not enabled.
```

Can we expand the `x` to `experimental:
```suggestion
    **Fixed** `istioctl experimental  wait` has undecipherable message when PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING is not enabled.
```